build.gradle.kts

```diff
@@ -98,6 +99,27 @@ tasks.test {
     maxParallelForks = 1
 }
 
+// Suppress warnings about incubating test suites feature
+@Suppress("UnstableApiUsage")
+testing {
```

Comment:
Once the test suite feature is stable it might be reasonable to convert the other test definitions to test suites as well.